404.php

```diff
@@ -22,44 +22,52 @@
 					<p><?php esc_html_e( 'Nothing was found at this location. Try searching, or check out the links below.', 'storefront' ); ?></p>
 
 					<?php
-					if ( is_woocommerce_activated() ) {
-						the_widget( 'WC_Widget_Product_Search' );
-					} else {
-						get_search_form();
-					}
+          echo '<section aria-label="Search">';
```

Comment:
This is a WordPress project and WordPress coding standards recommend tabs...
I just add a .editorconfig file in this project, so now you only need to install in vim: https://github.com/editorconfig/editorconfig-vim#readme